lbr_demos/lbr_demos_py/lbr_demos_py/asbr.py

```diff
@@ -0,0 +1,521 @@
+'''Collection of functions/types from the ASBR class taken in Spring 2022
+
+Most code is derived from the Matlab code that was written during that class,
+and is broadly attributable to Dr Alambeigi (who taught the course).
+'''
+
+import numpy as np
```

Comment:
is there a way to provide this functionality through a `pip` installable library?